doc/freeze/compress.md

```diff
@@ -79,3 +79,7 @@ Model compression, with little loss of accuracy, can greatly speed up MD inferen
 **Acceptable original model version**
 
 The model compression interface requires the version of deepmd-kit used in original model generation should be `2.0.0-alpha.0` or above. If one has a frozen 1.2 or 1.3 model, one can upgrade it through the `dp convert-from` interface.(eg: ```dp convert-from 1.2/1.3 -i old_frozen_model.pb -o new_frozen_model.pb```) 
+
+**Acceptable descriptor type**
+
+Note only descriptors with `se_e3_a` or `se_t` type are supported by the model compression feature.
```

Comment:
`se_e3_a` -> `se_e2_a`

`hybrid` mixed with above descriptors is also supported.

Reply:
Sure